### ISSUE TITLE
Issue with Initial Create DB Script

### ISSUE THREAD

**Issue by JoshDi** [metabrainz/musicbrainz-docker#156]
When running the **docker-compose run --rm musicbrainz createdb.sh -fetch** command, I get the following error:

```
..........
Resolving ftp.eu.metabrainz.org (ftp.eu.metabrainz.org)... 138.201.203.28
Connecting to ftp.eu.metabrainz.org (ftp.eu.metabrainz.org)|138.201.203.28|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /pub/musicbrainz/data/fullexport/20200530-001527 ... done.
==> SIZE mbdump-wikidocs.tar.bz2 ... 7079
==> PASV ... done.    ==> RETR mbdump-wikidocs.tar.bz2 ... done.
Length: 7079 (6.9K) (unauthoritative)

mbdump-wikidocs.tar.bz2       100%[==============================================>]   6.91K  --.-KB/s    in 0s

2020-06-02 13:59:27 (865 MB/s) - ‘/media/dbdump/mbdump-wikidocs.tar.bz2’ saved [7079]

/musicbrainz-server
Tue Jun  2 13:59:35 UTC 2020: Done fetching dump files.
found existing dumps
Tue Jun  2 13:59:35 2020 : InitDb.pl starting
ERROR:  schema "musicbrainz" already exists
ERROR:  schema "cover_art_archive" already exists
ERROR:  schema "documentation" already exists
ERROR:  schema "event_art_archive" already exists
ERROR:  schema "json_dump" already exists
ERROR:  schema "report" already exists
ERROR:  schema "sitemaps" already exists
ERROR:  schema "statistics" already exists
ERROR:  schema "wikidocs" already exists
Tue Jun  2 13:59:36 2020 : Installing extensions (Extensions.sql)
Tue Jun  2 13:59:36 2020 : Creating collations ... (CreateCollations.sql)
Tue Jun  2 13:59:36 2020 : psql:/musicbrainz-server/admin/sql/CreateCollations.sql:10: ERROR:  collation "musicbrainz" already exists
Error during CreateCollations.sql at /musicbrainz-server/admin/InitDb.pl line 117.
Tue Jun  2 13:59:36 2020 : InitDb.pl failed
```

To solve this problem, I need to run **docker-compose run --rm musicbrainz recreatedb.sh**

**Comment by JoshDi** [comment]
Does this have to do with this commit? [Fix amqp_setup after amqp 2.5.2 upgrade in dbfe94a](https://github.com/metabrainz/sir/commit/0424dc5659427a6d8fd4645e442800c1178d1ebf)

If I add **channel.open()** to the **sir/amqp/setup.py** file, I still get an error. 

```
root@HTPC-Xeon:/storage/musicbrainz-docker/admin# docker-compose exec indexer bash
root@e09866444883:/code# python -m sir amqp_setup
2020-06-02 15:40:23,949: Connecting to RabbitMQ
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/code/sir/__main__.py", line 127, in <module>
    main()
  File "/code/sir/__main__.py", line 123, in main
    func(args)
  File "sir/amqp/setup.py", line 22, in setup_rabbitmq
    conn = util.create_amqp_connection()
  File "sir/util.py", line 138, in create_amqp_connection
    virtual_host=cget("vhost"),
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 180, in __init__
    (10, 30),  # tune
  File "/usr/local/lib/python2.7/site-packages/amqp/abstract_channel.py", line 67, in wait
    self.channel_id, allowed_methods, timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 241, in _wait_method
    channel, method_sig, args, content = read_timeout(timeout)
  File "/usr/local/lib/python2.7/site-packages/amqp/connection.py", line 330, in read_timeout
    return self.method_reader.read_method()
  File "/usr/local/lib/python2.7/site-packages/amqp/method_framing.py", line 189, in read_method
    raise m
socket.error: [Errno 104] Connection reset by peer
```

**Comment by richsoni** [comment]
Worked like a charm
